### PR TITLE
Fix(check username):fix check username

### PR DESCRIPTION
**Pull request by Bankar-Swapnil**
### Functionality:
fix check username in profile page

### Solution:
Changes in edit-profile component

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
in profile page click on edit profile then change username and click on check availability button its shows loader first and then show username available message

### CODE REVIEW

File: src/app/body/profile/edit-profile/edit-profile.component.html [hunk 1]
```diff
@@ -35,7 +35,19 @@ <h2 class="modal-title " id="exampleModalLabel"><strong>Edit Profile</strong> </
                             <div class="col-6 align-middle"><a class="btn btn-primary" id="check_availability" (click)="checkAvailability()" style="background-color: #5c60ec;">check
                                 availability</a></div>
                         </div>
-                        <a *ngIf="userAvailable" class="btn btn-success">Username available</a>
+                        <a *ngIf="userAvailable" >
+                            <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+                                <symbol id="check-circle-fill" fill="currentColor" viewBox="0 0 16 16">
+                                  <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-3.97-3.03a.75.75 0 0 0-1.08.022L7.477 9.417 5.384 7.323a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-.01-1.05z"/>
+                                </symbol>
+                            </svg>
```
Comment (viv9k) [inline src/app/body/profile/edit-profile/edit-profile.component.html:43]: What is the use of this svg?

File: src/app/body/profile/edit-profile/edit-profile.component.html [hunk 2]
```diff
@@ -35,7 +35,19 @@ <h2 class="modal-title " id="exampleModalLabel"><strong>Edit Profile</strong> </
...
+                            <div class="alert alert-success d-flex align-items-center" role="alert">
+                                <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Success:"><use xlink:href="#check-circle-fill"/></svg>
```
Comment (viv9k) [inline src/app/body/profile/edit-profile/edit-profile.component.html:45]: And what is the use of this svg?

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

File: src/app/body/profile/edit-profile/edit-profile.component.html (hunk 1 above)
Comment (Bankar-Swapnil) [reply src/app/body/profile/edit-profile/edit-profile.component.html:43]: with the help of both two svg it shows an success alert with check sign logo

File: src/app/body/profile/edit-profile/edit-profile.component.html (hunk 2 above)
Comment (Bankar-Swapnil) [reply src/app/body/profile/edit-profile/edit-profile.component.html:45]: with the help of both two svg it shows an success alert with check sign logo

File: src/app/body/profile/edit-profile/edit-profile.component.html (hunk 1 above)
Comment (viv9k) [reply src/app/body/profile/edit-profile/edit-profile.component.html:43]: I think there are mat icons you can use to do it.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please use mat icon in place of svg

File: src/app/body/profile/edit-profile/edit-profile.component.html (hunk 1 above)
Comment (Bankar-Swapnil) [reply src/app/body/profile/edit-profile/edit-profile.component.html]: Changes done

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**